setup.cfg

```diff
@@ -0,0 +1,36 @@
+[metadata]
+name = qutip-cupy
+url = https://qutip.org
+description = CuPy backend for QuTiP
+long_description = file: README.md
+long_description_content_type = text/markdown; variant=GFM
+keywords = quantum, physics, dynamics, gpu, linear-algebra
+license = BSD 3-Clause License
+license_file = LICENSE.txt
+classifiers =
+    Development Status :: 2 - Pre-Alpha
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Topic :: Scientific/Engineering
+    Operating System :: MacOS
+    Operating System :: POSIX
+    Operating System :: Unix
+    Operating System :: Microsoft :: Windows
+author = Alexander Pitchford, Nathan Shammah, Shahnawaz Ahmed, Neill Lambert, Eric Giguere, Boxi Li, Jake Lishman, Simon Cross
+author_email = qutip-admin@googlegroups.com
+platforms = Linux, Mac OSX, Unix, Windows
+
+[options]
+package_dir =
+    =src
+packages = find:
+include_package_data = True
+install_requires =
+    qutip>=5.0.0.a0
```

Comment:
One minor thing that came up on today's meeting, Shouldn't this point to dev major at least for now?

Reply:
This is effectively the same thing; `master` is currently for the `4.x` series of QuTiP, while `dev.major` is the development branch for the `5.x` series, and the `VERSION` files in each branch fix that.  This is also a version specifier, so we want to be as lax as possible to allow the resolver to do its work as freely as we can.  If we pinned it to a specific git reference on `git+github.com:qutip/qutip.git@<ref>`, the resolver wouldn't allow you to use a local checkout of the repository, so it would be very difficult as soon as you wanted to do simultaneous development, or check if `qutip-cupy` works with changes you want to make in `qutip`.

Reply:
Great. I was getting confused with the git reference. Thank you.